### PR TITLE
Use through2 instead of event-stream

### PR DESCRIPTION
**Pull request by shinnn**
Same as #16 

---------------------------

I replaced [event-stream](https://github.com/dominictarr/event-stream) with [through2](https://github.com/rvagg/through2).

According to [the acceptance test suite for gulp plugins](https://github.com/gulpjs/acceptance), [it is not recommended that a gulp plugin depends on event-stream](https://github.com/gulpjs/acceptance#dependencies).

> **Reason** Old streams + bloated kitchen sink

Actually, this plugin doesn't use gazillions of event-stream's method.

### CODE REVIEW

**Comment by shinnn** [comment]
adamayres And, Could you give me write access for npm package? (`npm author add shinnn` in the right directory should do the trick)

File: index.js [hunk 1]
```diff
@@ -60,43 +17,95 @@ module.exports = function(opts, data, options){
...
         }
+
+        done(null, new Buffer(result));
       });
-    } else {
-      callback(null, file);
     }
 
-  }
+    var self = this;
```
Comment (adamayres) [inline index.js]: You might consider putting the `var` at the top of the function to mirror where it is hoisted too.

**Comment by adamayres** [comment]
Added as npm author.

**Comment by shinnn** [comment]
Hmm.... When I reset the commit, PR automatically closes at the same time and cannot re-open it.

@adamayres Is there any points to fix except for https://github.com/adamayres/gulp-wrap/pull/19#discussion_r22914804? If there isn't I'll fix the position of `var` and commit it to `master` directly.

**Comment by adamayres** [comment]
Strange, how are you resetting the commit? I'm fine if you just want to commit this change to master since it has already been reviewed.

**Comment by shinnn** [comment]
Done. ea680a31ea3d147051cfe9961b8c81813f78043b

**Pull request closed without merging**